Feed/index.html

```diff
@@ -48,84 +48,103 @@
                     <!-- Create Post Section -->
                     <div class="card mb-4 d-flex justify-content-center">
                         <div class="card-header">
-                            <h5>What's on your mind today?</h5>
-                        </div>
-                        <div class="card-body">
-                            <textarea class="form-control" rows="3" placeholder="Write your post..."></textarea>
-                        </div>
-                        <div class="card-footer">
-                            <button class="btn btn-success">Create Post</button>
+                            <h5>What's on your mind today, ?</h5>
                         </div>
+                        <form id="newPostForm">
+                            <div class="card-body">
+                                <textarea class="form-control" id="post-form-title" rows="1" placeholder="Title" required></textarea>
+                                <textarea class="form-control" id="post-form-body" rows="3" placeholder="Write your post..."></textarea>
+                                <textarea class="form-control" id="post-form-media" rows="1" placeholder="Media (add URL)"></textarea>
```

Comment:
Same thing here. Input element should be the correct element.

Reply:
fixed